### ISSUE TITLE
How to list start time and end time in the standard template timesheet

### ISSUE THREAD

**Issue by partoneoftwo** [kevinpapst/kimai2#1177]
Kimai version: Kimai 1.4
**User Story:**
As a user I would like to show my client the timestamps I am counting the time for each day.
This is evident when I use the standard PDF export, where in the left column, date, start time and date end time is nicely and neatly displayed.

The file I am trying to do this in is:
/templates/invoice/renderer/timesheet.html.twig

**Question:**
How do I get begin time and end time visible in the rows of the table showing on the timesheet invoice template?

**Comment by kevinpapst** [comment]
Do not customize existing templates/files, you will loose the changes with the next update.

All available keys are mentioned here: http://www.kimai.org/documentation/invoices.html

Grab ideas from the PDF template: https://github.com/kevinpapst/kimai2/blob/master/templates/export/renderer/pdf.html.twig#L146

**Comment by kevinpapst** [comment]
No feedback, closing

**Comment by partoneoftwo** [comment]
Hi there. Yes those were the two pieces of documentation which I had already used.
Sure they describe in detail and at length, but still do not help me help myself.
I simply don't understand how to do this.

Could you explain it? Re recap: What I aim to do is 

- to add a column in the invoice which contains START TIME and END TIME 
and 
- how do I populate each row in the invoice with START TIME and END TIME

Many thanks for your efforts on Kimai2. 
I have supported the project by buying one of your plugins.

**Comment by kevinpapst** [comment]
Hm, what did you already do? Which templates do you use? HTML, XLSX?

**Comment by kevinpapst** [comment]
I do understand that you are frustrated if it not works the way you would like it to do, but please keep in mind that I spent my free time to provide this software. 
If you donÄt provide feedback and include all details, I cannot help.

**Comment by partoneoftwo** [comment]
Hi there


>If you donÄt provide feedback and include all details, I cannot help.
I fully understand. I think the Github interface forces a quick close
period instead of leaving time for users to reply.

>Hm, what did you already do? Which templates do you use? HTML, XLSX?
The template I used was HTML.

When I tried to add the field, it seemed as if I needed to calculate the
time used. I was not able to create this calculated field.
So what I tried to do was to add two columns start_timestamp and
end_timestamp


Best regards,
Thomas
